### DISCUSSION TITLE
How to test non-story methods

### DISCUSSION THREAD

**Discussion by adchsm** [storybookjs/storybook#23751] [Help]
Hi there,

I've written a bunch of help util methods (in a separate utils.ts file) and additional tests (in a separate utils.test.ts file) to ensure they're working correctly. However, I can't seem to get the test runner to pick them up.

I've taken a look through the documentation and the config types and can't seem to spot any additional paths or config options to specify extra test files/paths.

Any help on this would be much appreciated!

Thanks,

Adam

**Comment by jestrickler** [comment]
What's the answer?
